### ISSUE TITLE
Symbolic link error upon install

### ISSUE THREAD

**Issue by nan0s7** [JLErvin/berry#12]
` ~/Sources/berry $ cp example/sxhkdrc /home/scott/.config/berry/sxhkdrc
cp: cannot stat 'example/sxhkdrc': Too many levels of symbolic links`

Looking at the sxhkdrc file; it says it is a broken link.

**Comment by JLErvin** [comment]
Very weird, not sure how that happened. Fixed in https://github.com/JLErvin/berry/commit/61a556cb6eacce43a6d31e3b6cb51b493cfe9767

**Comment by nan0s7** [comment]
Yep that fixed it.

Instead of opening something else; how do I know which monitor has which number associated with it for multi-monitor setups? I'm assuming monitor 1 is the main one? Or would that be monitor 0?

**Comment by JLErvin** [comment]
You can really use whatever utility you prefer. You can use `xrandr` and simply use `xrandr --listmonitors` to tell you which monitor is assigned to which screen. You could also use something like [disputils](https://github.com/tudurom/disputils). I'll add this to the README, however, as I'm sure other people will have this same question.

**Comment by nan0s7** [comment]
Ah okay, so `xrandr --listmonitors` gives:

`Monitors: 2
 0: +HDMI-2 2560/673x1080/284+1440+0  HDMI-2
 1: +HDMI-3 1440/408x900/255+0+0  HDMI-3`

However, I cannot get my second (monitor 1) monitor working no matter what I have as arguments for the `berryc save_monitor x x` option/command in the autostart file. The very first window I spawn (usually just urxvt) is place in the middle of both, but moving it once teleports it onto monitor 0. Sending monitors to the "saved" workspace for my second monitor doesn't show anything on the second monitor. I just switch to the workspace and the window is there on my first monitor.

I've tried `save_monitor` (1 2) (2 1) (3 1) (3 0) at least, and noticed nothing. As far as I can tell monitor 0 always starts on workspace 1, too.

Also, I have found a bug where if you send a window to a workspace, leaving the current workspace empty, the window will remain in the current workspace, still able to be interacted with (like in urxvt I can type, etc.) but none of the WM keyboard commands work on the window until I switch to the workspace I sent that window to. This issue doesn't occur when there is at least one window left on the original workspace.

I can't get the window moving using the mouse pointer at all now. Currently using sxhkd 0.5.5, but I also tried it on the latest version by mistake and on either version there was no responce; not even a glitch.

Also in the default sxhkdrc file, there is two assignments of the `super + o` shortcut for snapping the window to the left of the screen. :P

The quick fix you made to the window monocle resizing works though xD - although can I request a feature here? If you resize the window by any of those commands (snapping, fullscreen, monocle, etc.) that if you repeat the command that it reverts the window to its original dimensions? Or have a seperate command to restore original dimensions. I know I could use `window_resize w h` but then I'd have to have a seperate command for each of my windows.
